src/sections/Community/Handbook/about.js

```diff
@@ -10,6 +10,12 @@ import TOC from "../../../components/handbook-navigation/index";
 const IntroWrapper = styled.div`
     padding: 3rem 20rem;
     margin-top: -25rem; 
+    h1{
+      color: ${props => props.theme.darkJungleGreenColor};
```

Comment:
Why define the same CSS again? That's not needed.

Reply:
Applicable for font-size and font-weight as well

Reply:
alright..Got it! ...So only thing I need to keep is the margin-bottom property.